src/neuroconv/datainterfaces/ecephys/openephys/requirements.txt

```diff
@@ -1 +1 @@
-pyopenephys>=1.1.2
```

Comment:
Just to be clear, this still gets installed in the environment due to SpikeInterface: https://github.com/SpikeInterface/spikeextractors/blob/master/spikeextractors/extractors/openephysextractors/openephysextractors.py#L33

Reply:
Ah you're right, this should not be removed from the requirements.

Reply:
@weiglszonja Ah, my bad - that is the previous deprecated extractor

The current neo based one doesn't look like it depends on it at all